doc/rst/tools/mason/mason.rst

```diff
@@ -613,6 +819,79 @@ of its .toml file.
    ProjectB = "0.1.0"
 
 
+The Manifest File
+=================
+
+The ``Mason.toml`` manifest file is written in TOML(for more information see TOML section below).
```

Comment:
>TOML(for more information see TOML section below).

TOML (for more information see TOML section below).

--- 

Maybe make TOML a hyperlink to the TOML webpage?